apiserver/facades/agent/upgrader/unitupgrader.go

```diff
@@ -125,6 +118,82 @@ func (u *UnitUpgraderAPI) DesiredVersion(ctx context.Context, args params.Entiti
 	return params.VersionResults{Results: result}, nil
 }
 
+// SetTools is responsible for updating a a set of entities reported agent
+// version.
```

Comment:
Do we really need just one word on its own line? The line isn't too long such that it needs to wrap. YMMV.